### PR TITLE
Fix/event detail action

### PR DESCRIPTION
**Pull request by GagahGK**
| image | image<br> |
| ---- | ---- |

Changelog : 
* Fixes : Author actions buttons was shown for every logged in user
* Changed : Author action buttons now consist of edit,copy link, and join room
   * to delete the event author must go to edit event page first

### CODE REVIEW

File: app/(pages)/event/[eventID]/page.tsx [hunk 1]
```diff
@@ -133,6 +133,7 @@ export default async function Page({ params: { eventID } }: PageProps) {
         startTime={eventData.startTime}
         isPublished={eventData.isPublished}
         thumbnailUrl={eventData.thumbnailUrl}
+        createdBy={eventData.createdBy}
```
Comment (faiq-naufal) [inline app/(pages)/event/[eventID]/page.tsx:136]: I think you forget to pass `roomId` to `EventDetail` component here?

**Review by faiq-naufal — CHANGES_REQUESTED** [review]

**Comment by faiq-naufal** [comment]
@GagahGK  Some UI improvements, can you make the order of the action buttons UI when viewed as event creator align with the one viewed as guest user? 
1. Join webinar should be longer as CTA button. See the "Register to Attend" button
2. Move copy link button to the right side of "Join webinar" button. Copy link only displayed as icon when viewed on mobile
3. Move edit button to the right side of copy link button.

The order from left is Join webinar button - Copy link button - Edit button.

| image | image<br> |
| ---- | ---- |

**Comment by GagahGK** [comment]
>     1. Join webinar should be longer as CTA button. See the "Register to Attend" button
>     2. Move copy link button to the right side of "Join webinar" button. Copy link only displayed as icon when viewed on mobile

I don't think the copy Icon should be converted into icon only : 
1. Copy Link will still frequently accessed, making them small will not make them thumb friendly 
2. Marking them with different color is already enough to make them as CTA button since there's 3 action 
3. the edit button is small since it won't be accessed frequently than the other two

>     3. Move edit button to the right side of copy link button.
> The order from left is Join webinar button - Copy link button - Edit button. This order is for both mobile and desktop views.

Should we move it the otherway instead  ? , we need to give uniformity, that button with icons only is at the start of the action bar

or since there's only 2 action both of them could be the same size, primary CTA is already marked with red color

**Comment by faiq-naufal** [comment]
> I don't think the copy Icon should be converted into icon only : 
> 1. Copy Link will still frequently accessed, making them small will not make them thumb friendly 
> 2. Marking them with different color is already enough to make them as CTA button since there's 3 action 
> 3. the edit button is small since it won't be accessed frequently than the other two

@GagahGK You probably think the copy link button is important and frequently accessed. However, it's actually not that important. User can copy from the address bar directly in order to copy the event link. The copy link button is just a secondary button which helps the users copy the event link without having to copy from address bar. 

Have you seen [this design](https://www.figma.com/file/wjeG4AE78OXVZNxjWl09yn/inlive-room?type=design&node-id=256-2212&mode=design&t=uXUHVZL546MfkDXb-4)? In the design, the secondary button is icon only on mobile. The icon only button size is 40x40 which is already quite big size. Please follow to the actual design order.

Screenshot 2024-02-15 at 11 56 51

**Comment by GagahGK** [comment]
@faiq-naufal  updated

image

**Review by faiq-naufal — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**